src/deluge/model/drum/kit.cpp

```diff
@@ -652,6 +658,7 @@ void Kit::offerReceivedCCToLearnedParams(MIDIDevice* fromDevice, uint8_t channel
 	}
 }
 
+//not updated for midi follow, this seems dumb and is just left for backwards compatibility
```

Comment:
Pitch bend is a mod source in the synth engine post 4.0, so there's no reason to learn it to a parameter anymore. We could keep it so that it doesn't change the sound of old presets in case someone did use it